### DISCUSSION TITLE
He doesn't write me the email

### DISCUSSION THREAD

**Discussion by Juancafr** [charlesbel/Microsoft-Rewards-Farmer#454] [Q&A]
It opens my browser, it goes to the hotmail page and clicks next all the time, it doesn't write me the email, is this normal?

**Comment by The-Inspirer** [comment]
Also having the same issue

**Comment by chAosTwizt** [comment]
I had, and still sometime have the same issue. To get it to sign in I opened a new tab while its doing that and signed in manually, once signed in I refreshed the original tab and its logged in now. I still have this issue on one of the 4 accounts I use. I'm not sure why its not staying logged in, maybe I said no on that one when it prompted me to "Stay signed in?". When I used only one it was flawless.
